agent/qrexec-client-vm.c

```diff
@@ -132,10 +135,16 @@ int main(int argc, char **argv)
         start_local_process = 1;
     }
 
+    service_name = argv[optind + 1];
+
+    service_name_len_plus_one = strlen(service_name) + 1;
```

Comment:
All other code here seems to use length as a synonym for size (as supposed to length == strlen(x) == size - 1 as seen in some other code). So this should be just `service_name_len`.